generators/client/files-angular.js

```diff
@@ -47,6 +47,10 @@ const files = {
                 'postcss.config.js',
                 { file: 'webpack/logo-jhipster.png', method: 'copy' }
             ]
+        },
+        {
+            condition: generator => !generator.skipCommitHook,
+            templates: [{ file: '../.huskyrc', method: 'copy', noEjs: true, renameTo: () => '.huskyrc' }]
```

Comment:
this is weird, why do you have `../` in file name? and you dont have to rename it unless there is a change

Reply:
This should be fine `{ file: '.huskyrc', method: 'copy' }` and also where is the actual file? did you forget to commit it?

Reply:
The `../` in the filename is because I wanted to have only one file in the `templates` folder instead of one in `templates/react` and `templates/angular`.

So for angular, files are created with `this.writeFilesToDisk(files, this, false, this.fetchFromInstalledJHipster('client/templates/angular'));` so the `../` is to go back to `templates`. And then the renameTo is to avoid having the file in the parent directory of the project.

It's a little hacky but that's a way I found to avoid duplicating `.huskyrc` (or in `package.json`).

I see two other ways:
- Copy the file in the `write` method before the switch:

```javascript
// Public API method used by the getter and also by Blueprints
_writing() {
    return {
        write() {
            // COPY .HUSKYRC HERE
            switch (this.clientFramework) {
                case 'react':
                    return writeReactFiles.call(this, useBlueprint);
                default:
                    return writeAngularFiles.call(this, useBlueprint);
            }
        }
    };
}
```

- Call `this.writeFilesToDisk('.huskyrc', this, false, this.fetchFromInstalledJHipster('client/templates'));` in `file-angular.js` and `file-react.js`.

Which way you think is the best ?

Reply:
Please duplicate and put the files in respective folders for now, so that we don't have weird behaviors when using blueprints and stuff. We have some ideas for moving common code out but that's a bigger story for later